pkg/component/kubernetes/apiserver/apiserver_test.go

```diff
@@ -1274,6 +1288,64 @@ resources:
 					Expect(kapi.Deploy(ctx)).To(MatchError(ContainSubstring("either the name of an existing secret or both certificate and private key must be provided for TLS SNI config")))
 				})
 			})
+			It("should create the istio tls secrets for feature gate IstioTLSTermination", func() {
+				DeferCleanup(test.WithFeatureGate(features.DefaultFeatureGate, features.IstioTLSTermination, true))
+				sm = &fakeSecretsManagerWithoutCurrent{sm}
+				kapi = New(kubernetesInterface, namespace, sm, Values{
+					Values: apiserver.Values{
+						RuntimeVersion: runtimeVersion,
+					},
+					Version: version,
+					SNI: SNIConfig{
+						IstioIngressGatewayNamespace: "istio-ingress",
+						TLS:                          []TLSSNIConfig{},
+					},
+				})
+
+				Expect(kapi.Deploy(ctx)).To(Succeed())
+
+				expectedTlsSecret := &corev1.Secret{
+					ObjectMeta: metav1.ObjectMeta{Name: namespace + "-kube-apiserver-tls", Namespace: "istio-ingress"},
+					Data: map[string][]byte{
+						"cacert": nil,
+						"cert":   nil,
+						"key":    nil,
+					},
+				}
+				expectedCaSecret := &corev1.Secret{
+					ObjectMeta: metav1.ObjectMeta{Name: namespace + "-kube-apiserver-ca", Namespace: "istio-ingress"},
+					Data: map[string][]byte{
+						"cacert": nil,
+						"cert":   nil,
+						"key":    nil,
+					},
+				}
+
+				managedResource := &resourcesv1alpha1.ManagedResource{ObjectMeta: metav1.ObjectMeta{Name: "istio-tls-secrets", Namespace: namespace}}
+				Expect(c.Get(ctx, client.ObjectKeyFromObject(managedResource), managedResource)).To(Succeed())
+				Expect(managedResource).To(consistOf(expectedTlsSecret, expectedCaSecret))
+			})
+			It("should clean up the istio tls secrets after disabling feature gate IstioTLSTermination", func() {
```

Comment:
Please leave a newline between the tests.

```suggestion

			It("should clean up the istio tls secrets after disabling feature gate IstioTLSTermination", func() {
```